src/main.ts

```diff
@@ -22,13 +24,18 @@ export async function run() {
     dotnet.disableTelemetry();
     dotnet.disableWelcomeMessage();
 
-    await cakeTool.install(toolsDir, version);
+    if (!csprojPath) {
```

Comment:
**suggestion:** positive conditions are easier to read than negative ones. I think this `if` statement should be inverted.

Reply:
Looks like this one was resolved with latest rebase  
  
 

![](https://github.com/cake-build/cake-action/assets/1905010/5c3f08ea-e832-4549-96bc-8ed3cd5392b3)